### ISSUE TITLE
mocha watch mode causes new snapshots to be written on every test suite run

### ISSUE THREAD

**Issue by davidnormo** [suchipi/chai-jest-snapshot#12]
When I run mocha in watch mode, every new test run causes new snapshots to be written to `.snap` files. So `.snap` files grow over time even when no changes are made.

Running mocha manually every time doesn't add new ones, as expected.

mocha v3.2.0
chai v3.5.0
chai-jest-snapshot v0.3.0

I'll try investigating a little, let me know if I can provide more information.

**Comment by suchipi** [comment]
To clarify; the counter increasing is what's causing the issue?

**Comment by davidnormo** [comment]
Yep

**Comment by lgandecki** [comment]
yes, I can confirm. This issue makes this basically unusuable :-( reruning tests manually is not a reasonable long-term solution for a big project with a lot of tests.

**Comment by suchipi** [comment]
This issue has been fixed in 0.4.0, but you will need to add some new code to your test setup file in order to make things work correctly:
```js
before(function() {
  chaiJestSnapshot.resetSnapshotRegistry();
});
```
The documentation in the [README](https://github.com/suchipi/chai-jest-snapshot/blob/master/README.md#mocha-configuration-mode-recommended-for-mocha-users) has been updated to reflect this.
